diff -ru apache_1.3.42.orig/src/support/htdigest.c apache_1.3.42/src/support/htdigest.c
--- apache_1.3.42.orig/src/support/htdigest.c	2006-07-12 10:16:05.000000000 +0200
+++ apache_1.3.42/src/support/htdigest.c	2014-07-19 19:39:34.230609062 +0200
@@ -71,7 +71,7 @@
     while ((line[y++] = line[x++]));
 }
 
-static int getline(char *s, int n, FILE *f)
+static int _getline(char *s, int n, FILE *f)
 {
     register int i = 0;
 
@@ -158,7 +158,7 @@
 {
     static char line[MAX_STRING_LEN];
 
-    while (!(getline(line, MAX_STRING_LEN, source))) {  
+    while (!(_getline(line, MAX_STRING_LEN, source))) {  
 	putline(target, line);
     }
 }
@@ -216,7 +216,7 @@
     ap_cpystrn(realm, argv[2], sizeof(realm));
 
     found = 0;
-    while (!(getline(line, MAX_STRING_LEN, f))) {
+    while (!(_getline(line, MAX_STRING_LEN, f))) {
 	if (found || (line[0] == '#') || (!line[0])) {
 	    putline(tfp, line);
 	    continue;
diff -ru apache_1.3.42.orig/src/support/htpasswd.c apache_1.3.42/src/support/htpasswd.c
--- apache_1.3.42.orig/src/support/htpasswd.c	2006-07-12 10:16:05.000000000 +0200
+++ apache_1.3.42/src/support/htpasswd.c	2014-07-19 19:39:40.682556577 +0200
@@ -98,7 +98,7 @@
  * Get a line of input from the user, not including any terminating
  * newline.
  */
-static int getline(char *s, int n, FILE *f)
+static int _getline(char *s, int n, FILE *f)
 {
     register int i = 0;
 
@@ -547,7 +547,7 @@
 	char scratch[MAX_STRING_LEN];
 
 	fpw = fopen(pwfilename, "r");
-	while (! (getline(line, sizeof(line), fpw))) {
+	while (! (_getline(line, sizeof(line), fpw))) {
 	    char *colon;
 
 	    if ((line[0] == '#') || (line[0] == '\0')) {
diff -ru apache_1.3.42.orig/src/support/logresolve.c apache_1.3.42/src/support/logresolve.c
--- apache_1.3.42.orig/src/support/logresolve.c	2008-10-06 16:29:12.000000000 +0200
+++ apache_1.3.42/src/support/logresolve.c	2014-07-19 19:40:11.930613279 +0200
@@ -71,7 +71,7 @@
 #endif /* !MPE && !WIN32*/
 
 static void cgethost(struct in_addr ipnum, char *string, int check);
-static int getline(char *s, int n);
+static int _getline(char *s, int n);
 static void stats(FILE *output);
 
 
@@ -278,7 +278,7 @@
  * gets a line from stdin
  */
 
-static int getline (char *s, int n)
+static int _getline (char *s, int n)
 {
     char *cp;
 
@@ -326,7 +326,7 @@
     for (i = 0; i < MAX_ERR + 2; i++)
 	errors[i] = 0;
 
-    while (getline(line, MAXLINE)) {
+    while (_getline(line, MAXLINE)) {
 	if (line[0] == '\0')
 	    continue;
 	entries++;
